### ISSUE TITLE
Accessibility Testing

### ISSUE THREAD

**Issue by Buwujiu** [Pan-Canadian-Genome-Library/daco#431]
## Descriptions
Use Axe to perform accessibility testing on DACO. 
Manual testing on keyboard navigation. 
## Acceptance Criteria
**1. Dan to fix all critical violations by March. 12**

**Comment by patrickdsantos** [comment]
Really wonderful work team, the website is looking great! Most critical issues from AXE have been resolved along with keyboard functionality. :)

Just a couple of minor updates:
1. The red colour we are using for error messaging does not have enough contrast for AA accessibility, please use: #E7131A instead
2. I can't use the TAB button on the keyboard to access the footer. 

Thanks team! 

@Buwujiu @demariadaniel
